packages/p2p/src/utils/orders.js

```diff
@@ -0,0 +1,152 @@
+            labels.instructions = localize('Your instructions');
+            labels.result_string = localize('You sold {{offered_amount}} {{offered_currency}}', {
+                offered_amount: order_details.amount_display,
+                offered_currency: order_details.account_currency,
+            });
+        } else {
+            labels.other_party_role = localize('Seller');
+            labels.left_send_or_receive = localize('Send');
+            labels.right_send_or_receive = localize('Receive');
+            labels.payment_details = localize("Seller's payment details");
+            labels.contact_details = localize("Seller's contact details");
+            labels.instructions = localize("Seller's instructions");
+            labels.result_string = localize("You've received {{offered_amount}} {{offered_currency}}", {
+                offered_amount: order_details.amount_display,
+                offered_currency: order_details.account_currency,
+            });
+        }
+    } else if (is_my_ad) {
+        labels.other_party_role = localize('Seller');
+        labels.left_send_or_receive = localize('Send');
+        labels.right_send_or_receive = localize('Receive');
+        labels.payment_details = localize("Seller's payment details");
+        labels.contact_details = localize("Seller's contact details");
+        labels.instructions = localize('Your instructions');
+        labels.result_string = localize("You've received {{offered_amount}} {{offered_currency}}", {
+            offered_amount: order_details.amount_display,
+            offered_currency: order_details.account_currency,
+        });
+    } else {
+        labels.other_party_role = localize('Buyer');
+        labels.left_send_or_receive = localize('Receive');
+        labels.right_send_or_receive = localize('Send');
+        labels.payment_details = localize('Your payment details');
+        labels.contact_details = localize('Your contact details');
+        labels.instructions = localize("Buyer's instructions");
+        labels.result_string = localize('You sold {{offered_amount}} {{offered_currency}}', {
+            offered_amount: order_details.amount_display,
+            offered_currency: order_details.account_currency,
+        });
+    }
+
+    return labels;
+};
+
+export const getExtendedOrderDetails = (order_details, loginid) => {
+    const { advert_details, advertiser_details, client_details, status, type } = order_details;
+
+    const is_buy_ad = type === 'buy';
+    const is_sell_ad = type === 'sell';
+
+    const is_buyer_cancelled_order = status === 'cancelled';
+    const is_buyer_confirmed_order = status === 'buyer-confirmed';
+    const is_completed_order = status === 'completed';
+    const is_expired_order = status === 'timed-out';
+    const is_pending_order = status === 'pending';
+    const is_refunded_order = status === 'refunded';
+
+    const is_my_ad = loginid === advertiser_details.loginid;
```

Comment:
Just curious, why `loginid` and not `advertiser_id` ? 🤔

Reply:
Both `loginid` and `advertiser_id` could be used but I found `loginid` to be less ambigous.